Drug-related mortality.R

```diff
@@ -32,20 +32,20 @@ drug_deaths_male_02_05 <- read.csv(paste0(data_folder, "Received Data/drugs_deat
                                 setNames(tolower(names(.)))
 
 # NRS file for drug deaths - match to SQL query by rdno_entry_no_yr
-drug_deaths_NRS <- read.spss(paste0(data_folder, "Received Data/NRS_DRDs0620.sav"), 
+drug_deaths_NRS <- read.spss(paste0(data_folder, "Received Data/NRS_DRDs0621.sav"), 
                 to.data.frame=TRUE, use.value.labels=FALSE) %>%
                 setNames(tolower(names(.)))
 
-# SQL query for drug deaths 2006-2020
+# SQL query for drug deaths 2006-2021
 # Select drug specific deaths from SMRA - the ICD10 codes have been removed from query as it was not finding all the deaths in the NRS extract (Sep21)
 # Select only deaths for scottish residents (COR=XS)
```

Comment:
it doesn't actually look like we have a selection on scottish residents so remove this comment?

Reply:
Oh yeah, will do.

Reply:
Lee Barnsdale has confirmed that the figures should replicate the NRS figures, so he has approved the change in methodology.